src/state/index.ts

```diff
@@ -61,12 +62,14 @@ const store = configureStore({
     elasticFarm,
     tokenPrices,
     topTokens,
+    routeApi: routeApi.reducer,
   },
   middleware: getDefaultMiddleware =>
     getDefaultMiddleware({ thunk: true, immutableCheck: false, serializableCheck: false })
       .concat(save({ states: PERSISTED_KEYS, debounce: 100 }))
       .concat(geckoTerminalApi.middleware)
-      .concat(annoucementApi.middleware),
+      .concat(annoucementApi.middleware)
```

Comment:
Why is it here? Why is it not added before

Reply:
Only`,` is the difference =))

Reply:
:a-the-a: